### PR TITLE
Add a link to factory_bot in the documentation

### PR DESCRIPTION
**Pull request by ydah**
This PR is add a link to factory_bot in the documentation.

______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [-] Added tests.
- [x] Updated documentation.
- [-] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 = RuboCop factory_bot
 
-factory_bot-specific analysis for your projects, as an extension to
+https://www.rubydoc.info/gems/factory_bot[factory_bot]-specific analysis for your projects, as an extension to
```
Comment (pirj) [inline docs/modules/ROOT/pages/index.adoc:3]: My usual entry point to FB is https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md
The rubydoc page is far less useful.
WDYT?

Comment (ydah) [reply docs/modules/ROOT/pages/index.adoc]: Indeed it is. I updated this PR.

**Comment by pirj** [comment]
Thank you!
Please merge at will, GH misbehaves for me from the mobile.

**Comment by pirj** [comment]
Thank you!
Please merge at will, GH misbehaves for me from the mobile.

**Pull request merged**